contracts/clearing/BrokerVault.sol

```diff
@@ -98,23 +108,20 @@ contract BrokerVault is IVault2, ISettle, Ownable2Step, ReentrancyGuard {
 		INitroTypes.RecoveredVariablePart calldata candidate
```

Comment:
Add `nonReentrant` modifier to `settle` function too

Reply:
391f764